Get Single prop from props

radenkovic opened the pull request:
Allows use of `get()`, also supports lodash jsonPath: `get('theme.colorPrimary')`

- [x] get()
- [x] tests
- [x] Readme
- [x] Formatting/linting

radenkovic committed:
Get Single prop from props

sergioramos commented:
This is great, thank you 🙏 

1) Feel free to choose a different name, I like all your suggestions 😉;
2) If you use something like `lodash.get`, you get a battle tested handler of the attribute getter.

radenkovic committed:
use lodash fp/get and rename property to get

radenkovic committed:
update typescript definition

radenkovic commented:
> This is great, thank you 
> 
> 1. Feel free to choose a different name, I like all your suggestions wink;
> 2. If you use something like `lodash.get`, you get a battle tested handler of the attribute getter.

I just updated it to use lodash/fp get, at first I was hesitant to include lodash. 

- Property is renamed to `get()`
- Added ts definition

I feel this is good to go!

callmeberzerker commented:
Go for the win? :)
